### DISCUSSION TITLE
Knx ultimate node cannot catch some event from knx bus

### DISCUSSION THREAD

**Discussion by freeranwang** [Supergiovane/node-red-contrib-knx-ultimate#309] [General]
**Salutation (i'll not respond to users that doesn’t wrote at least "Hello")**
Hi,
**Describe the bug**
In the Home Assistant system, I have integrated KNX along with Node-RED, using the KNX-Ultimate node. I am retrieving data through a KNX/IP gateway. I have recently observed that the group monitor in the KNX integration of Home Assistant, as well as the KNX-Ultimate node, are unable to capture all bus messages (compared to what I see in the group monitor of ETS). It appears that Home Assistant's KNX integration is specifically unable to capture messages from KNX-Ultimate, and KNX-Ultimate does not respond to many group address read messages (these messages can be captured in both ETS and Home Assistant's KNX integration group monitor)

Another peculiar issue is that in ETS, I am reading the source address of messages sent by KNX-Ultimate as 15.15.242, but the address I have configured in the KNX-Ultimate gateway settings is not this one. I'm not entirely clear on the underlying principle behind this discrepancy.

**Comment by Supergiovane** [comment]
Hi
I cannot really help you, as i don't use Home Assistant, but as far as i can tell, such issues arises when you have some UDP storming protection on your switches, or if you have updated the firmware of your network appliance.
For example, everytime i upgrade my Unifi stuffs, i need to reboot them, because i lost UDP communications.
You should ask on HA forum or, if you know German language, you could ask the knx-user-forum.de

**Reply by freeranwang** [reply]
thank you so much! i'll do some tests with my network appliance.
